src/structures/MessageAttachment.js

```diff
@@ -43,6 +43,24 @@ class MessageAttachment {
     return this;
   }
 
+  /**
+   * Sets whether this attachment is a spoiler
+   * @param {boolean} [spoiler=true] Whether the attachment should be marked as a spoiler
+   * @returns {MessageAttachment} This attachment
+   */
+  setSpoiler(spoiler = true) {
+    if (spoiler === this.spoiler) return this;
+
+    if (!spoiler) {
+      while (this.spoiler) {
+        this.name.slice('SPOILER_'.length);
```

Comment:
is it worth making a constant for this?

eg just
```js
const SPOILER_PREFIX = 'SPOILER_';
```
at the top or exported in constants

Reply:
We should be fine without.